README.md

```diff
@@ -1058,6 +1069,17 @@ hierarchy:
 This would give precedence to all Kafo-managed parameter values, but for any
 others, would check for values per OS family, followed by a `common.yaml` file.
 
+The scenario name is also available as a fact which can be used to provide
+scenario specific overrides for unmanaged modules.
+
+```yaml
+hierarchy:
+  - name: "Kafo Answers"
+    path: "%{facts.kafo.scenario.answer_file}"
+  - name: "Scenario defaults"
+    path: "scenario/%{facts.kafo.scenario.name}.yaml"
```

Comment:
`kafo.scenario.name` in the above example is `Foreman`, but I'd expect to see `foreman.yaml` (lower-case F) for the filename?

Reply:
So right now `name` is the human readable name. I'm thinking about introducing a more usable one like `id` which would match the one passed on the command line.